core/modules/views/includes/view.inc

```diff
@@ -1891,6 +1891,7 @@ class view {
       }
     }
     $clone->display = $displays;
+    $clone->storage = 1;
```

Comment:
Use proper constant `VIEWS_STORAGE_NORMAL`, in order to make the code self-explicit